content/10.diagnostics.md

```diff
@@ -241,130 +247,127 @@ The authors generated an appropriate electric field gradient using a selective i
+RT-qPCR identified 147 samples as positive and 192 as negative.
+They estimated the overall sensitivity and specificity to be 61.9% and 99.0%, respectively.
+Sensitivity increased with higher C~t~.
+This study also assessed the performance of the ELISA test under different conditions in order to evaluate how robust it would be to the challenges of testing in real-world settings globally.
+Higher sensitivity was achieved for samples that were stored under ideal conditions (immediate placement in -80° C).
+Therefore, while immediate access to laboratory equipment is an advantage, it is not strictly necessary for ELISA to detect the antigen.
 
 #### Limitations of Molecular Tests
 
 Tests that identify SARS-CoV-2 using nucleic-acid-based technologies will identify only individuals with current infections and are not appropriate for identifying individuals who have recovered from a previous infection.
-Within this category, different types of tests have different limitations.
-For example, PCR-based test can be highly sensitive, but in high-throughput settings they can show several problems:
-First of all, there is a risk of false-negative responses, which can present a significant problem to large-scale testing.
+Among molecular tests, different technologies have different sensitivities and specificities.
+In general, specificity is high, and even then, the public health reprecussions of a false positive can generally be mitigated with follow-up testing.
+On the other hand, a test's sensitivity, which indicates the risk of a false-negative response, can pose significant challenge to large-scale testing.
+False negatives are a significant concern for several reason.
+Importantly, clinical reports indicate that it is imperative to exercise caution when interpreting the results of molecular tests for SARS-CoV-2 because negative results do not necessarily mean a patient is virus-free [@doi:10.1128/JCM.00297-20].
 To reduce occurrence of false negatives, correct execution of the analysis is crucial [@doi:10.1038/d41587-020-00002-2].
-Additionally, the emerging nature of the COVID-19 pandemic has introduced some challenges related to Uncertainty surrounding interactions between SARS-CoV-2 and its human hosts.
-For example, viral shedding kinetics are still not well understood, but are expected to introduce a significant effect of timing of sample collection on test results [@doi:10.1038/d41587-020-00002-2].
-Similarly, the type of specimen could also influence outcomes, as it is not clear which clinical samples are best for detecting the virus [@doi:10.1038/d41587-020-00002-2].
-There are also significant practical and logistical concerns.
+Additionally, PCR-based tests can remain positive for a much longer time than the virus is likely to be actively replicating [@doi:10.1056/NEJMcp2117115], raising concerns about their informativeness after the acute phase of the disease.
```

Comment:
```suggestion
Additionally, PCR-based tests can remain positive for a much longer time than the virus is likely to be actively replicating [@doi:10.1056/NEJMcp2117115], raising concerns about their informativeness after the acute phase of the disease.
Hence why the CDC has advised individuals who suspect they have been re-infected with SARS-CoV-2 to avoid using diagnostic tests within 90 days of receiving a previous positive test [@url:https://www.cdc.gov/coronavirus/2019-ncov/your-health/quarantine-isolation.html].
```